bin/build-docs.js

```diff
@@ -310,4 +311,13 @@ async function compileDocs(services) {
     });
 }
 
+function splitErrors(str) {
+    const important = [];
+    const ignored = [];
+    for (const line of str.split('\n').map(s => s.trim()).filter(s => s.length)) {
```

Comment:
An alternative way to write this would be:
```javascript
const isImportant = line => !line.includes('WARNING: duplicate function description');
const nonEmptyLines = str.split('\n').map(s => s.trim()).filter(s => s.length);
const [important, ignored] = _.partition(nonEmptyLines, isImportant);
```
Either is fine and you don't need to make any changes but I thought I would mention it since I think `_.partition` is a little more legible. :)

Reply:
Yeah, forgot I used the weird ternary thing. I just refactored to use partition, and I pulled out the ignore patterns into a list so we can just add to that in the future.